config.json

```diff
@@ -579,7 +579,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "wordy",
-      "topics": null,
+      "topics": [
+        "strings",
+        "parsing",
+        "pattern_matching",
+        "transforming"
```

Comment:
Great suggestions! :) I would maybe add `logic` and `integers` as well. And maybe `regular_expressions`? What do you think?

Reply:
And also `exception_handling` since the tests expect an exception to be thrown :)

Reply:
Good points, I'll update the PR shortly